tardis/workflows/standard_simulation_solver.py

```diff
@@ -0,0 +1,318 @@
+import logging
+from pathlib import Path
+
+import numpy as np
+from astropy import units as u
+
+from tardis import constants as const
+from tardis.io.atom_data.base import AtomData
+from tardis.model import SimulationState
+from tardis.plasma.standard_plasmas import assemble_plasma
+from tardis.simulation.convergence import ConvergenceSolver
+from tardis.spectrum.base import SpectrumSolver
+from tardis.spectrum.formal_integral import FormalIntegrator
+from tardis.transport.montecarlo.base import MonteCarloTransportSolver
+
+# logging support
+logger = logging.getLogger(__name__)
+
+
+class StandardSimulationSolver:
+    def __init__(self, configuration):
```

Comment:
If we want this to replace the full run_tardis etc it should probably be able to take in a configuration path and convert it to a configuration object. Probably as the only option.